.vscode/launch.json

```diff
@@ -7,7 +7,8 @@
             "program": "${workspaceFolder:sourcekit-lsp}/.build/debug/sourcekit-lsp",
             "args": [],
             "cwd": "${workspaceFolder:sourcekit-lsp}",
-            "preLaunchTask": "swift: Build Debug sourcekit-lsp"
+            "preLaunchTask": "swift: Build Debug sourcekit-lsp",
+            "sourceLanguages": ["swift"]
```

Comment:
I also set the sourceLanguage to swift so the CodeLLDB on swift throw breakpoint is available.